scripts/tokenomics-emulation/config.json

```diff
@@ -28,6 +28,7 @@
     "cere_sc_address": "5C8VXSSmizvyUoKaTkyYTsqrchj3rAgjNHbNPJANhZkXqSfR",
     "gas_limit": -1,
     "smart_contract_cere_token_amount_default": 0,
+    "format_balance_decimals": 10,
```

Comment:
You can use decimals which we have. It is no sense to have 2 decimals: 1 for formatting and 1 for chain spec.

Reply:
using the decimals